.circleci/config.yml

```diff
@@ -6,15 +6,19 @@ common-steps:
       paths:
         - "*"
 
-  - &removevirtualenv
-    run:
-      name: Removes the upstream virtualenv from the original container image
-      command: sudo pip uninstall virtualenv -y
-
   - &installdeps
     run:
       name: Install Debian packaging dependencies
-      command: make install-deps
+      command: |
+        apt-get update && apt-get install -y make sudo
+        make install-deps
+
+  - &installtestdeps
+    run:
+      name: Install test dependencies
+      command: |
+        sudo apt-get install reprotest faketime -y --no-install-recommends
```

Comment:
I'm curious about the use of `sudo` here and for the `sudo sed` commands -- is that needed here?

Reply:
No, it's not needed anymore. I think I had left it alone just to reduce the diff, but let's get rid of it.